modules/osint/domain_reputation.py

```diff
@@ -0,0 +1,156 @@
+"""
+OWASP Maryam!
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import re
+import concurrent.futures
+import json
+import requests
+
+meta = {
+    'name': 'Domain Reputation',
+    'author': 'Kunal Khandelwal',
+    'version': '0.1',
+    'description': 'Check domain reputation with different sources and provide a summary of combined results',
+    'sources': ('barracudacentral', 'mxtoolbox', 'multirbl'),
+    'options': (
+        ('domain', None, False, 'Domain name without https?://', '-d', 'store', str),
+        ('engines', ['barracudacentral', 'multirbl', 'mxtoolbox'], False, 'Search engine names(default=[barracudacentral,multirbl])'
```

Comment:
you should add mxtoolbox in the default description as well

Reply:
Yes, I lost this.

Reply:
Done